src/cli-options.ts

```diff
@@ -448,3 +451,47 @@ function parseRemainingArgs(
     extraArgs,
   };
 }
+
+const DEFAULT_WATCH_STRATEGY = {strategy: 'event'} as const;
+const DEFAULT_WATCH_INTERVAL = 500;
+
+/**
+ * Interpret the WIREIT_WATCH_* environment variables.
+ */
+function readWatchConfigFromEnv(): Options['watch'] {
+  switch (process.env['WIREIT_WATCH_STRATEGY']) {
+    case 'event':
+    case '':
+    case undefined: {
+      return DEFAULT_WATCH_STRATEGY;
+    }
+    case 'poll': {
+      let interval = DEFAULT_WATCH_INTERVAL;
+      const intervalStr = process.env['WIREIT_WATCH_POLL_MS'];
+      if (intervalStr) {
+        const parsed = Number(intervalStr);
+        if (Number.isNaN(parsed) || parsed <= 0) {
+          console.error(
```

Comment:
If this is continuing to operate with the fallback value, should this be `console.warn()` instead?

Reply:
Both go to stderr, but in some configurations warnings might get filtered out? I guess because it's definitely something wrong, I think it should be an error to maximize the chance of somebody seeing it.